GNUmakefile

```diff
@@ -220,14 +220,17 @@ ifeq (${openmp},1)
         LDFLAGS  += -fopenmp
     endif
 else
-    slate_src += src/stubs/openmp_stubs.cc
+    ${error ERROR: unsupported `openmp=${openmp}`. OpenMP is required.}
```

Comment:
We could completely remove `${openmp}`, but that would create more conflicts with #146.